src/istio/values/config-admin.yaml

```diff
@@ -1,6 +1,7 @@
 name: admin
 domain: "admin.###ZARF_VAR_DOMAIN###"
 tls:
+  supportTLSV1_2: false # set to true to enable TLSv1.2 support
```

Comment:
nit: since this aligns with the default value (`false`) we don't really need it in all these config values files.